pylutron/__init__.py

```diff
@@ -289,12 +289,17 @@ def _parse_area(self, area_xml):
   def _parse_output(self, output_xml):
     """Parses an output, which is generally a switch controlling a set of
     lights/outlets, etc."""
-    output = Output(self._lutron,
-                    name=output_xml.get('Name'),
-                    watts=int(output_xml.get('Wattage')),
-                    output_type=output_xml.get('OutputType'),
-                    integration_id=int(output_xml.get('IntegrationID')),
-                    uuid=output_xml.get('UUID'))
+    output_type = output_xml.get('OutputType')
+    output_cls = Output
```

Comment:
I'm not a big fan of flinging around class types like this. Almost as hacky but looks cleaner to me would be to create a dict with those args and then just **kwargs it to Output() or Shade() explicitly, thoughts? One person's clean is another person's ugly I suppose. This list will grow eventually as we add the other controllers from other PRs and there are motors, etc here.

Reply:
I could do something like that sure.